back/spec/services/side_fx_user_spec.rb

```diff
@@ -82,20 +82,27 @@
     end
   end
 
-  describe 'after_block' do
-    it "logs a 'blocked' action job when a user is blocked" do
-      user.update(block_start_at: Time.now)
-      expect { service.after_block(user, current_user) }
-        .to have_enqueued_job(LogActivityJob)
-        .with(user, 'blocked', current_user, user.updated_at.to_i, payload: { block_reason: user.block_reason })
+  include_context 'when user_blocking duration is 90 days' do
```

Comment:
This context is included just to get tests green on CI, but will not removed by changes in #4252